src/List/NonEmpty/Zipper.elm

```diff
@@ -284,6 +284,28 @@ insertAfter a (Zipper b f n) =
     Zipper b f (a :: n)
 
 
+{-| Insert a new value at the start, without changing focus.
+
+Note: This is linear complexity, meaning that if the number
+of items before the current focus doubles, the time this function
+takes also doubles.
+
+    fromConsList [2, 3] (4, [5])
+    |> insertStart 1
+    |> toList
+    --> [1,2,3,4,5]
+
+    fromConsList [2, 3] (4, [5])
+    |> insertStart 1
+    |> current
+    --> 4
+
+-}
+insertStart : a -> Zipper a -> Zipper a
+insertStart a (Zipper b f n) =
+    Zipper (b ++ [ a ]) f n
```

Comment:
I was thinking maybe it would make more sense to do:

```elm
prepend : List a -> Zipper a -> Zipper a
```

What do you think?

Reply:
Sounds good to me!